content/members.md

```diff
@@ -1,8 +1,41 @@
 +++
-title = "SONIX connected members"
+title = "SONIX Friends"
+description = "SONIX Friends - The member section"
+keywords = ["sonix friends", "connected networks", "membership", "members"]
 +++
 
-# Members
+# Become one of our SONIX Friends
+
+## What Are SONIX Friends?
+
+At SONIX, we call our members "SONIX Friends". These are the networks, organizations, and individuals who connect to our Internet Exchange Points (IXPs) or that just supports our mission of improving Sweden’s internet connectivity. Being a SONIX Friend is about more than just connecting—it’s about contributing to a better-connected, more sustainable future.
+
+---
+
+## We're Powering a Better-Connected Sweden
+
+We are incredibly proud of and grateful to our SONIX Friends. They play a vital role in shaping Sweden’s internet infrastructure by:
+- Providing crucial infrastructure and network expertise.
+- Supporting our mission through contributions.
+- Engaging with the community to promote collaboration and innovation.
+
+Together, we are building a faster, more affordable, and sustainable internet for everyone.
+
+---
+
+## Become a SONIX Friend!
+
+Interested in joining our growing community? Becoming a SONIX Friend means being part of a collaborative effort to improve connectivity across Sweden. 
+
+Learn more about partnership opportunities and how you can get involved:
+- [Join the Network](/join/network-partner)
+- [Contribute](/contribute/)
+
+---
+
+## Connected Networks
+
+Explore the networks already connected to SONIX:
 
 <div id="table-container"></div>
-<script src="/members-table.js"></script>
```

Comment:
The table which should be the main focus of this page ends up too far down. Most of the copy on this page should be removed. Maybe part of it can be moved to the FAQ or an About page.